hack/build

```diff
@@ -25,6 +25,7 @@ engine |\
         ${_EXPERIMENTAL_DAGGER_GPU_SUPPORT:+--gpu-support} |\
     start \
         --name=$CONTAINER \
+        --debug \
```

Comment:
I think adding this by default is causing the sdk dev jobs to fail, e.g. https://github.com/dagger/dagger/actions/runs/13730718938/job/38407111073?pr=9809

pretty sure 6060 is just conflicting with something... I'm not 100% sure what it's conflict with off the top of my head, but maybe it's as easy as just changing the port to 6061.

Worst case we don't have to leave in this `--debug` but it would be nice since I personally also manually add this a lot

Reply:
noted, won't commit this blindly. tbh i would prefer if we could use the default port here if we do commit this.